server/databaseConnection.js

```diff
@@ -22,6 +22,7 @@ const refreshDb = (db) => {
     db.collection('Questions').deleteMany({}),
     db.collection('Employees').deleteMany({}),
     db.collection('Admins').deleteMany({}),
+    db.dropDatabase(),
```

Comment:
do we definitely want to be doing this? It seems quite destructive

Reply:
Agreed, removed.